### PR TITLE
Get out of here old mixin

### PR DESCRIPTION
**Pull request by DFurnes**
Removes the old navigation styling mixins and some old body class overrides.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss [hunk 1]
```diff
@@ -2,64 +2,6 @@
...
-        color: #fff !important;  // @TDOD: blarg!
-      }
-
-      .hamburger {
-        color: #fff !important;  // @TDOD: blarg!
-      }
-    }
-  }
-
-  #user-profile-form {
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss:41]: Did the form styling get removed too?

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss:41]: Using the `user--profile--edit` page template now: aef4cc6

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss:41]: hokay!

**Comment by weerd** [comment]
:dancer: :+1:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss:41]: screen shot 2014-06-27 at 12 20 45 pm

**Pull request merged**